lib/apiservers/engine/backends/backends.go

```diff
@@ -290,6 +290,7 @@ func syncContainerCache() error {
 			errs = append(errs, err.Error())
 		}
 	}
+
```

Comment:
This seemed really cluttered with the if statement pushed right against the for loop.